loadbalancer/src/main/java/com/boundary/dropwizard/consul/loadbalancer/AbstractLBFactory.java

```diff
@@ -72,20 +72,22 @@ protected ServiceHealthCache buildCache(Environment env, HealthClient healthClie
                 getWatchSeconds()
         );
 
-        env.lifecycle().manage(new Managed() {
-            @Override
-            public void start() throws Exception {
-                cache.start();
-                if (!cache.awaitInitialized(10, TimeUnit.SECONDS)) {
-                    throw new Exception("load balancer init timeout");
+        if (env != null) {
```

Comment:
@gjesse This should suffice to meet the requirement of not needing a dropwizard Environment to create a LoadBalancer instance.